packages/clay-toolbar/src/index.tsx

```diff
@@ -52,6 +58,9 @@ const ClayToolbar: React.FunctionComponent<IProps> & {
 		{
 			'component-tbar': !subnav,
 			'subnav-tbar': !!subnav,
+			[`tbar-${
+				displayType === 'light' ? displayType : `${displayType}-l2`
```

Comment:
What happens if you don't pass in `displayType`? I envision users wanting to pass in their own custom modifier class, maybe we can give 2 options `light` and `dark`, but don't output anything if displayType has no value.

Also there are two placeholders for dark, `.dark-d1` and `.dark-l2`. Lexicon has no definitions for these so, maybe we don't need a dark option at all.

Reply:
If `displayType` isn't provided, the toolbar's background is transparent.

> Lexicon has no definitions for these so, maybe we don't need a dark option at all.

Sure, I don't have any reason to argue, I'll omit the `displayType` prop and add a boolean `light` prop that adds the `tbar-light` class.